test/e2e-tests.sh

```diff
@@ -31,7 +31,7 @@ source $(dirname $0)/../vendor/github.com/knative/test-infra/scripts/e2e-tests.s
 # Build kn before integration tests, so we fail fast in case of error.
 function cluster_setup() {
   header "Building client"
-  go build -v -mod=vendor ./cmd/... || return 1
+  $(dirname $0)/../hack/build.sh -u || return 1
```

Comment:
@rhuss Do you think it is more straightforward to add export GO111MODULE=on above Line 34?
The command $(dirname $0)/../hack/build.sh -u goes thru 🕸️  Update 🚧 Compile
🧪  Test🧹  Format 📖 Docs, but we only need compile here.

Reply:
We can change to `build.sh -f` which just compiles.